operator-framework-core/src/test/java/io/javaoperatorsdk/operator/processing/event/source/AbstractEventSourceTest.java

```diff
@@ -0,0 +1,46 @@
+package io.javaoperatorsdk.operator.processing.event.source;
+
+import org.junit.jupiter.api.AfterEach;
+
+import io.javaoperatorsdk.operator.processing.event.EventHandler;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class AbstractEventSourceTest<S extends EventSource, T extends EventHandler> {
```

Comment:
This should be a Junit extension, as for Integration tests. I think there should be no inheritance in unit tests.

Reply:
Feel free to change that.